### PR TITLE
Add nodejs 22

### PR DESCRIPTION
**Pull request by omBratteng**
Adds nodejs 20, which is the new LTS version

### CODE REVIEW

File: .github/workflows/image-check.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ concurrency:
 jobs:
   diff:
     runs-on: distroless-ci-large-ubuntu-20.04 # custom runner most compatible with debian 11
+    permissions:
+      pull-requests: write
```
Comment (omBratteng) [inline .github/workflows/image-check.yaml:16]: I added this as the workflow only gets `read` permissions if the author of a PR is outside the organisation. But the `peter-evans/create-or-update-comment` action needs `write` access to be able to update the PR.

Comment (loosebazooka) [reply .github/workflows/image-check.yaml:16]: I don't think we should expand permissions. Don't worry about the diff image check.

Comment (omBratteng) [reply .github/workflows/image-check.yaml]: reverted

**Comment by smorimoto** [comment]
Can we move forward with this PR?

**Comment by loosebazooka** [comment]
Oh yeah I'll look when I'm back at work tomorrow

**Comment by smorimoto** [comment]
Thanks a lot!

**Review by loosebazooka — APPROVED** [review]

**Comment by loosebazooka** [comment]
Thanks again @omBratteng for your commitment to the distroless node images!

**Pull request merged**